compiler/src/iree/compiler/Codegen/Common/TransformDialectStrategies.cpp

```diff
@@ -651,9 +775,12 @@ static bool matchGPUReduction(linalg::LinalgOp op, TileSizesListType &tileSizes,
                      // Only single combiner over 32 bits for now due to
                      // reduction distribution.
                      .output(0, ElementTypeBitWidth(32))
-                     .output(0, SingleCombinerReduction());
+                     .output(0, SingleCombinerReduction())
+                     .result(0, HasAnyUse(), maybeTrailingEltwise);
   if (!matchPattern(op, pattern)) return false;
 
+  hasTrailingEltwise = maybeTrailingEltwise.succeeded();
+
```

Comment:
nit below:
```
tilesSizes.workgroup.assign({numLoops, 1});
tileSizes.fillSubgroup.assign({1, 0, 0});  // I think subgroup is the right name for this in IREE .. not 100% sure
tileSizes.genericSubgroup.assign({1, 1, 0});
```

This would force `TileSizesListType` to be its own type, valid only as a `ReductionStratefyTileSizesListType` but it would also be more explicit.

Reply:
This is literally what https://github.com/iree-org/iree/pull/11222 is doing and not in scope of this PR.